src/main/java/uk/gov/ch/model/update/OverseasEntityData.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ch.model.update;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class OverseasEntityData {
+
+    @Id
+    @Column(name = "corporate_body_id")
+    @JsonIgnore
+    private Long corporateBodyId;
+
+    @Column(name = "email_address")
+    private String emailAddress;
+
+    public Long getCorporateBodyId() {return corporateBodyId;}
```

Comment:
minor - let's be consistent with inline or expanded functions for the getters/setters